app/views/play/CampaignView.coco.coffee

```diff
@@ -1054,6 +1054,8 @@ module.exports = class CampaignView extends RootView
     if new Date(me.get('dateCreated')) < new Date('2021-09-21')
       defaultAccess = 'all'
     access = me.getExperimentValue 'home-content', defaultAccess
+    if me.showChinaResourceInfo() or me.get('country') is 'japan'
```

Comment:
dont need this change in .ozar?

Reply:
actually TIL we dont use .ozar play campaignview at all.   oz uses /ozaria/site/play ...

Reply:
I think it's just used in campaign editor, which is why it wasn't deleted/fully merged yet. Not relevant for this change, though.